build/ci/linux/runmtests.sh

```diff
@@ -7,17 +7,18 @@ df -k .
 
 . ./../musescore_environment.sh
 
-cd build.release/mtest
+cd build.debug/mtest
 
-# vnc is the only tested platform plugin that allows to run
-# mscore executable in the used Travis environment.
-export QT_QPA_PLATFORM=vnc
-export ASAN_OPTIONS=detect_leaks=0
+# run the mtests in "minimal" platform for headless systems
+# enable fonts handling
+export QT_QPA_PLATFORM=minimal:enable_fonts
+# if AddressSanitizer was used, disable leak detection
+export ASAN_OPTIONS=detect_leaks=0:new_delete_type_mismatch=0
 
 make -j2
 
 df -k .
 
-xvfb-run -a ctest -j2 --output-on-failure
+ctest -j2 --output-on-failure
 
-df -k .
\ No newline at end of file
+df -k .
```

Comment:
Not quite sure what the purpose of these is (I guess some debugging output), but a) IIRC `-k` is the default as per POSIX, and b) Linux' df's `-h' might be more helpful, 'human readable'?

Reply:
We had a problem with free space.
I added this for monitoring
I will change to `-h`, thanks.

Reply:
https://github.com/musescore/MuseScore/pull/6687

Reply:
Didn't notice those back then ;-)